src/style.css

```
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.big-icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.small-icon {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

move font family to `html` selector

```suggestion
  font-family: inherit;
```